localization/languages/pt-BR.json

```diff
@@ -5,7 +5,7 @@
         /* Context menu items
         these are the items displayed in the menu when you right-click on a page
         */
-        "pictureInPicture": null, //missing translation
+        "pictureInPicture": "Picture in picture", //missing translation
```

Comment:
I had the same question on #927 actually: is there not a Portuguese translation for this?